rand_distr/src/pert.rs

```diff
@@ -12,12 +12,20 @@ use core::fmt;
 use num_traits::Float;
 use rand::Rng;
 
-/// The PERT distribution.
+/// The PERT distribution `PERT(min, max, mode, shape)`.
 ///
 /// Similar to the [`Triangular`] distribution, the PERT distribution is
 /// parameterised by a range and a mode within that range. Unlike the
 /// [`Triangular`] distribution, the probability density function of the PERT
 /// distribution is smooth, with a configurable weighting around the mode.
+/// If left unspecified, the default value of `shape` is `4.0`.
```

Comment:
We don't need to mention this: the `new` constructor does. It's slightly confusing also since Rust does not support default parameters.